catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/DumpCommand.java

```diff
@@ -265,7 +279,24 @@ protected Object executeWithSubject() throws Exception {
     }
 
     if (StringUtils.isNotBlank(zipFileName)) {
-      createZip(catalog, queryRequest, dirPath + zipFileName, resultCount);
+      File outputFile = new File(dirPath + zipFileName);
+      createZip(catalog, queryRequest, outputFile, resultCount);
+
+      String alias = System.getProperty(SystemBaseUrl.EXTERNAL_HOST);
+      String password = System.getProperty("javax.net.ssl.keyStorePassword");
+
+      try (InputStream inputStream = new FileInputStream(outputFile)) {
+        byte[] signature = signer.createDigitalSignature(inputStream, alias, password);
+
+        if (signature == null) {
+          console.println("An error occurred while signing export");
```

Comment:
❓ Do we only need to print that we could not sign it? Do we need to take any action here (specifically, do we need to fail or anything of that sort if we can't sign?)

imo- i think printing an error and leaving the file as is - is probably okay. If we really wanted to be in their face about it we could rename the dump zip to `<name>_UNSIGNED.zip` but that may be overkill 🤷‍♂️